DataManagementSystem/Service/IRODSStorageElementHandler.py

```diff
@@ -206,7 +207,7 @@ def __getFileStat( self, path ):
       resultDict['Migrated'] = 0
       resultDict['Lost'] = 0
       resultDict['Unavailable'] = 0
-      resultDict['Mode'] = 0755
+      resultDict['Mode'] = 0o755
```

Comment:
Is this a new notation ? And why not using the stat module as you did somewhere else ?